zboxcore/sdk/allocation.go

```diff
@@ -2139,7 +2139,7 @@ func (a *Allocation) getConsensuses() (fullConsensus, consensusThreshold int) {
 		return a.DataShards, a.DataShards
 	}
 
-	return a.DataShards + a.ParityShards, a.DataShards + 1
+	return a.DataShards + a.ParityShards, a.DataShards
```

Comment:
What's the reason for this change?
Minimum Data+1 number of shards are required to recover the full data as far as I remember. @lpoli please review this change.